ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -134,12 +136,60 @@ export default class ExperimentalTab extends PureComponent {
     );
   }
 
+  renderEIP1559V2EnabledToggle() {
+    if (!process.env.EIP_1559_V2) {
+      return null;
+    }
+    const { t } = this.context;
+    const { eip1559V2Enabled, setEIP1559V2Enabled } = this.props;
+
+    return (
+      <div className="settings-page__content-row">
+        <div className="settings-page__content-item">
+          <span>{t('enableEIP1559V2')}</span>
+          <div className="settings-page__content-description">
+            {t('enableEIP1559V2Description', [
+              <a
+                key="eip_page_link"
+                href="https://metamask.io/1559.html"
```

Comment:
This link is for our 1559 explainer. Should it be included even though it doesn't have any information specifically pertaining to the changes included in 1559 V2 UI enhancements?

Reply:
Let me check with Kevin about this.

Reply:
Draft of v2 enhancements: https://docs.google.com/document/d/1Osh2UJ6Yfqc6V3jkH1jeX5fAg35_ug-P5Fb_G_TPoAY/edit#heading=h.95tpxiusoa6e

Reply:
So is the plan here to update the site but maintain the same link? And we just haven't pushed those updates to the site yet?

Reply:
yes that is right @adonesky1